### PR TITLE
Guides for migrating data

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#270

This adds a guide for creating a task to migrations data from an InfluxDB Cloud bucket to an OSS bucket in time-based batches.

Related to https://github.com/influxdata/community-templates/pull/300

### CODE REVIEW

File: content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+your InfluxDB Cloud bucket, the task will return an error similar to:
+
+```
+unauthorized access
+```
+
+**Possible solutions**:
+- Ensure the API token has read access to your InfluxDB Cloud bucket.
+- Generate a new InfluxDB Cloud API token with read access to the bucket you
+  want to migration. Then update the `INFLUXDB_CLOUD_TOKEN` secret in your
```
Comment (jstirnaman) [inline content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md:356]:
```suggestion
  want to migrate. Then, update the `INFLUXDB_CLOUD_TOKEN` secret in your
```

File: content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+  InfluxDB OSS instance with the new token.
+
+### Query timeout
+The InfluxDB Cloud query timeout is 90 seconds. If it takes longer than this to
+return the data from the batche's time range, the query will time out and the
```
Comment (jstirnaman) [inline content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md:361]:
```suggestion
return the data from the batch interval, the query will time out and the
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+    batchInterval: 1h,
+    batchBucket: "migration",
+    cloudHost: "https://cloud2.influxdata.com",
+    cloudOrg: "example-cloud-org",
+    cloudToken: secrets.get(key: "INFLUXDB_CLOUD_TOKEN"),
+    cloudBucket: "example-cloud-bucket",
+    ossBucket: "example-oss-bucket",
+}
+
+// batchRange dynamically returns a record with start and stop properties for
```
Comment (jstirnaman) [inline content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md:114]: 👍

File: content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+<!----------------------- BEGIN Determine batch interval ---------------------->
+{{% expand "Determine your batch interval" %}}
+
+The `migration.batchInterval` setting controls the time range queried by each batch.
+The "density" of the data in your InfluxDB Cloud bucket and your InfluxDB Cloud
+organization's [rate limits and quotas](/influxdb/cloud/account-management/limits/)
+determine what your batch interval should be.
+
+For example, if you are migrating data collected from hundreds of sensors with
```
Comment (jstirnaman) [inline content/influxdb/v2.1/write-data/migrate-cloud-to-oss.md:258]:
```suggestion
For example, if you're migrating data collected from hundreds of sensors with
```

**Comment by sanderson** [comment]
@jstirnaman A little experimentation, yes 😄. Thanks for the review!

**Review by sunbryely-influxdata — APPROVED** [review]

**Pull request merged**